bin/template-update.rdf

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?><RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+<Description about="urn:mozilla:extension:devtools@mozilla.org">
+<em:updates><Seq><li>
+  <Description>
+    <em:version>@@ADDON_VERSION@@</em:version>
+    <em:targetApplication>
+      <Description>
+        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+        <em:minVersion>26.0a1</em:minVersion>
+        <em:maxVersion>27.0</em:maxVersion>
```

Comment:
Yes, it is just wrong I copy pasted that from adbhelper.
Here is a in-production one that look more correct:
https://archive.mozilla.org/pub/labs/valence/win32/update.rdf